terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,92 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                = "filing-create" # this must match the stack name the service deploys into
+  name_prefix               = "${local.stack_name}-${var.environment}"
+  service_name              = "extensions-web"
+  container_port            = "3000" # default node port required here until prod docker container is built allowing port change via env var
+  docker_repo               = "extensions-web"
+  lb_listener_rule_priority = 91
+  lb_listener_paths         = ["/extensions/*"]
+  healthcheck_path          = "/extensions/healthcheck" #healthcheck path for extensions web
+  healthcheck_matcher       = "200"
+
+  service_secrets           = jsondecode(data.vault_generic_secret.service_secrets.data_json)
+
+  parameter_store_secrets    = {
+    "vpc_name"                  = local.service_secrets["vpc_name"]
+    "chs_api_key"               = local.service_secrets["chs_api_key"]
+    "internal_api_url"          = local.service_secrets["internal_api_url"]
+    "oauth2_auth_uri"           = local.service_secrets["oauth2_auth_uri"]
+    "oauth2_redirect_uri"       = local.service_secrets["oauth2_redirect_uri"]
+    "account_test_url"          = local.service_secrets["account_test_url"]
+    "account_url"               = local.service_secrets["account_url"]
+    "cache_server"              = local.service_secrets["cache_server"]
+    "oauth2_client_id"          = local.service_secrets["oauth2_client_id"]
+    "oauth2_client_secret"      = local.service_secrets["oauth2_client_secret"]
+  }
+
+  vpc_name                  = local.service_secrets["vpc_name"]
+  chs_api_key               = local.service_secrets["chs_api_key"]
+  internal_api_url          = local.service_secrets["internal_api_url"]
+  oauth2_auth_uri           = local.service_secrets["oauth2_auth_uri"]
+  oauth2_redirect_uri       = local.service_secrets["oauth2_redirect_uri"]
+  account_test_url          = local.service_secrets["account_test_url"]
+  account_url               = local.service_secrets["account_url"]
+  cache_server              = local.service_secrets["cache_server"]
+  oauth2_client_id          = local.service_secrets["oauth2_client_id"]
+  oauth2_client_secret      = local.service_secrets["oauth2_client_secret"]
+
+  # create a map of secret name => secret arn to pass into ecs service module
+  # using the trimprefix function to remove the prefixed path from the secret name
+  secrets_arn_map = {
+    for sec in data.aws_ssm_parameter.secret:
+      trimprefix(sec.name, "/${local.name_prefix}/") => sec.arn
+  }
+
+  service_secrets_arn_map = {
+    for sec in module.secrets.secrets:
+      trimprefix(sec.name, "/${local.service_name}-${var.environment}/") => sec.arn
+  }
+
+  # TODO: task_secrets don't seem to correspond with 'parameter_store_secrets'. What is the difference?
```

Comment:
I think we added this TODO comment when we did the PR for OE Web, but presumably that's been answered now and this comment can go (in both places)? Need to check.

Reply:
Removed this comment.

Reply:
Also removed oauth and account test from locals pss.